src/contracts/help-text/help-text-id.ts

```diff
@@ -1,3 +1,5 @@
 export enum HelpTextId {
   ScriptTaskTokenUsageHelpText = 'script-task-token-usage',
+  ExternalTaskTokenUsage = 'external-task-token-usage',
+  UserTaskUsage = 'usertask-usage',
```

Comment:
Ich würde die Namen hier konsistent halten, also entweder so:
```suggestion
  ScriptTaskTokenUsageHelpText = 'script-task-token-usage',
  ExternalTaskTokenUsageHelpText = 'external-task-token-usage',
  UserTaskUsageHelpText = 'usertask-usage',
```
oder so:
```suggestion
  ScriptTaskTokenUsage = 'script-task-token-usage',
  ExternalTaskTokenUsage = 'external-task-token-usage',
  UserTaskUsage = 'usertask-usage',
```